optaplanner-persistence/optaplanner-persistence-jaxb/src/test/java/org/optaplanner/persistence/jaxb/api/score/buildin/bendable/BendableScoreJaxbXmlJsonAdapterTest.java

```diff
@@ -21,17 +21,30 @@
 
 import org.junit.Test;
 import org.optaplanner.core.api.score.buildin.bendable.BendableScore;
-import org.optaplanner.persistence.jaxb.api.score.AbstractScoreJaxbXmlAdapterTest;
+import org.optaplanner.persistence.jaxb.api.score.AbstractScoreJaxbXmlJsonAdapterTest;
 
-public class BendableScoreJaxbXmlAdapterTest extends AbstractScoreJaxbXmlAdapterTest {
+public class BendableScoreJaxbXmlJsonAdapterTest extends AbstractScoreJaxbXmlJsonAdapterTest {
 
     @Test
-    public void serializeAndDeserialize() {
-        assertSerializeAndDeserialize(null, new TestBendableScoreWrapper(null));
+    public void serializeAndDeserializeXML() {
+        assertSerializeAndDeserializeXML(null, new TestBendableScoreWrapper(null));
+
         BendableScore score = BendableScore.of(new int[]{1000, 200}, new int[]{34});
-        assertSerializeAndDeserialize(score, new TestBendableScoreWrapper(score));
+        assertSerializeAndDeserializeXML(score, new TestBendableScoreWrapper(score));
+
+        score = BendableScore.ofUninitialized(-7, new int[]{1000, 200}, new int[]{34});
+        assertSerializeAndDeserializeXML(score, new TestBendableScoreWrapper(score));
+    }
+
+    @Test
+    public void serializeAndDeserializeJson() {
+        assertSerializeAndDeserializeJson(null, new TestBendableScoreWrapper(null));
+
+        BendableScore score = BendableScore.of(new int[]{1000, 200}, new int[]{34});
+        assertSerializeAndDeserializeJson(score, new TestBendableScoreWrapper(score));
+
         score = BendableScore.ofUninitialized(-7, new int[]{1000, 200}, new int[]{34});
-        assertSerializeAndDeserialize(score, new TestBendableScoreWrapper(score));
+        assertSerializeAndDeserializeJson(score, new TestBendableScoreWrapper(score));
     }
```

Comment:
By splitting up serializeAndDeserialize() into serializeAndDeserializeXML() and serializeAndDeserializeJson() for every Test class, we're create a lot of duplication, for all of the 10+ score classes. Note how both methods are indentical except for the assertSerializeAndDeserialize*() call.

Let's be practical. Let's keep it as 1 method and keep using only assertSerializeAndDeserialize(), but let's have that - in the superclass - call assertSerializeAndDeserializeXML() and assertSerializeAndDeserializeJson().
That will heavily reduce maintenance once we mess with scores again (like the time I added initScore, all these tests had to be refactored).